### DISCUSSION TITLE
Relative dates seen off by one year in GitHub site

### DISCUSSION THREAD

**Discussion by pyrtsa** [community/community#84193] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## Steps to reproduce

Browse around https://github.com/ today and see how past dates from roughly _N_ years ago are displayed in relative time format.

### Expected outcome

I'd expect dates from approximately 1 year ago to display as `1 year ago`. Similarly, two years ago as `2 years ago`.

### Actual outcome

Dates from one year ago—give or take a few days—display as `2 years ago` right now. Simlarly, two years ago as `3 years ago`. (See screenshots below)

## Screenshots

It's a bit before noon on 2024-01-05 (UTC+02) right now, but GitHub UI renders events from one year ago as `2 years ago`:

<img width="238" alt="Screenshot 2024-01-05 at 11 13 47" src="https://github.com/community/community/assets/212795/1dd28c26-76b0-4e69-9162-b39817cc7c49">
<img width="256" alt="Screenshot 2024-01-05 at 11 24 09" src="https://github.com/community/community/assets/212795/bea35ac9-4fda-46ca-a8d3-71d70f30c4d3">

----

Likewise, here's a date from a _little bit under_ 2 years ago seen as `3 years ago`:

<img width="207" alt="Screenshot 2024-01-05 at 11 35 40" src="https://github.com/community/community/assets/212795/d47616f9-8f38-4d8e-a556-852aafc9b288">

----

In fact, I'd also expect this other example from 2023-11-01—which was 2 months, 4 days ago—to be rendered as `2 months ago`, not `3 months ago`:

<img width="203" alt="Screenshot 2024-01-05 at 11 24 51" src="https://github.com/community/community/assets/212795/2f0405bc-d84a-47d0-a72e-1c3879288f98">

**Comment by levelad** [comment]
Just noticed something similar, file change date shows "2 days ago", but if you hover over it, it shows almost 1 year ago.

![GitHub_time_off](https://github.com/user-attachments/assets/c254f870-133e-4576-af94-bb0bf607f7b9)

**Comment by L-Clouds** [comment]
Was just about to make a report of the same thing, but did a search first and found this as well as two other subsequent reports of the same thing: #148704 and #150544
